Documentation/compatibility/winforms-accessibility-changes-472.md

```diff
@@ -35,15 +35,16 @@ In order for the application to benefit from these changes, it must run on the .
+```    
+Note that to opt in to the accessibility features added in .NET Framework 4.7.2, you must also opt in to accessibility features of .NET Framework 4.7.1 as well.
+Applications that target the .NET Framework 4.7.2 or later and want to preserve the legacy accessibility behavior can opt in to the use of legacy accessibility features by explicitly setting this AppContext switch to `true`.
 
 __Use of OS-defined colors in High Contrast themes__
-- The drop down arrow arrow of the `T:System.Windows.Forms.ToolStripDropDownButton` now has uses now use OS-defined colors in High Contrast theme.
-- `T:System.Windows.Forms.Button`, `T:System.Windows.Forms.RadioButton` and `T:System.Windows.Forms.CheckBox` controls with `P:System.Windows.Forms.Control.FlatStyle` set to <xref:System.Windows.Forms.FlatStyle.Flat?displayProperty=nameWithType> or <xref:System.Windows.Forms.FlatStyle.Popup?displayProperty=nameWithType> now use OS-defined colors in High Contrast theme when selected. Previously, text and background colors were not contrasting and were hard to read.
-- Controls contained within a `System.Windows.Forms.GroupBox` which has its `P:System.Windows.Forms.Enabled` property set to _false_ will now use OS-defined colors in High Contrast theme.
-- `T:System.Windows.Forms.ToolStripButton`, `T:System.Windows.Forms.ToolStripComboBox`, and `T:System.Windows.Forms.ToolStripDropDownButton` controls have an increased luminosity contrast ratio in High Contrast Mode. 
-- `T:System.Windows.Forms.DataGridViewLinkColumn` will by default use OS-defined colors in High Contrast mode for the `P:System.Windows.Forms.DataGridViewLinkColumn.LinkColor` property.
+- The drop down arrow arrow of the <xref:System.Windows.Forms.ToolStripDropDownButton> now uses OS-defined colors in High Contrast theme.
+- <xref:System.Windows.Forms.Button>, <xref:System.Windows.Forms.RadioButton> and <xref:System.Windows.Forms.CheckBox> controls with <xref:System.Windows.Forms.Control.FlatStyle?displayProperty=nameWithType> set to <xref:System.Windows.Forms.FlatStyle.Flat?displayProperty=nameWithType> or <xref:System.Windows.Forms.FlatStyle.Popup?displayProperty=nameWithType> now use OS-defined colors in High Contrast theme when selected. Previously, text and background colors were not contrasting and were hard to read.
+- Controls contained within a <xref:System.Windows.Forms.GroupBox> that has its <xref:System.Windows.Forms.Control.Enabled> property set to 'false' will now use OS-defined colors in High Contrast theme.
+- <xref:System.Windows.Forms.ToolStripButton>, <xref:System.Windows.Forms.ToolStripComboBox>, and <xref:System.Windows.Forms.ToolStripDropDownButton> controls have an increased luminosity contrast ratio in High Contrast Mode. 
+- <xref:System.Windows.Forms.DataGridViewLinkButton> will by default use OS-defined colors in High Contrast mode for the <xref:System.Windows.Forms.DataGridViewLinkButton.LinkColordisplayProperty=nameWithType> property.

```

Comment:
Do these xrefs show up properly? When I click 'view' it doesn't look right. I would think we would need ` to style things as code and [] () format for links since this is markdown.

Reply:
Or is the xref format necessary for when these are ingested into docs.microsoft.com?

Reply:
The XRef's are needed for the ingestion. @rpetrusha , can you make sure I did them all right? I was confused as to when to add the ?displayProperty=nameWithType on to it.